specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-10-02-preview/examples/ManagedClustersAssociate_CRG.json

```diff
@@ -0,0 +1,256 @@
+{
+  "parameters": {
+    "api-version": "2022-10-02-preview",
+    "subscriptionId": "subid1",
+    "resourceGroupName": "rg1",
+    "resourceName": "clustername1",
+    "parameters": {
+      "location": "location1",
+      "tags": {
+        "tier": "production",
+        "archv2": ""
+      },
+      "sku": {
+        "name": "Basic",
+        "tier": "Free"
+      },
+      "properties": {
+        "kubernetesVersion": "",
+        "dnsPrefix": "dnsprefix1",
+        "agentPoolProfiles": [
+          {
+            "name": "nodepool1",
+            "count": 3,
+            "vmSize": "Standard_DS2_v2",
+            "osType": "Linux",
+            "type": "VirtualMachineScaleSets",
+            "enableNodePublicIP": true,
+            "mode": "System",
+            "capacityReservationGroupID": "/subscriptions/subid1/resourcegroups/rg1/providers/Microsoft.Compute/capacityReservationGroups/crg1"
```

Comment:
>capacityReservationGroupID [](http://example.com/codeflow?start=13&length=26)

most places I see 'Id' in property names its capitalized as 'Id', not 'ID'

Reply:
This property was added in **2021-11-01-preview**. Reformatting the name would cause a breaking change.